ui/app/styles/helper-classes/layout.scss

```diff
@@ -68,7 +68,7 @@
 }
 
 .is-medium-width {
-  width: $desktop / 3;
+  width: calc($desktop / 3);
```

Comment:
this class is used in the `certificate-card.hbs` so decided to tackle this deprecation warning while here
<img width="634" alt="Screenshot 2023-12-04 at 3 47 00 PM" src="https://github.com/hashicorp/vault/assets/68122737/38dcc0da-a65b-4b46-a91f-e640dc96b35b">